plugins/plugins.pro

```diff
@@ -6,9 +6,9 @@
 TEMPLATE = subdirs
 
 CONFIG += debug_and_release
-SUBDIRS = link
+SUBDIRS = l4d2 link
```

Comment:
As in: this will break on macOS, FreeBSD, etc., where the required headers for plugins are not available.